### ISSUE TITLE
Add package uploader/maintainers to the Package metadata API

### ISSUE THREAD

**Issue by Duppils** [pypa/warehouse#9978]
**What's the problem this feature will solve?**
 
Help identify trustworthy package uploaders. Currently, the package metadata API https://pypi.org/project/{package_name}/json returns the repository maintainers, but not the package maintainers. Accessing package uploader/maintainer can help build credibility to the package or expose risks.

**Describe the solution you'd like**
 
Package maintainer is added to the API. If the package maintainers' historic contributions could be added to this or a separate API, that would help identify trustworthy packages.

**Additional context**
 
Home-brewed or forked packages, which should not inherit credibility, such as https://pypi.org/project/f-ask/. This package at a glance (incorrectly) looks to be owned by the pallets team, which has a different level of trust associated with it. This was just an example, please do not negatively affect whoever uploaded it. I do not wish to check if it was a malicious typo-squat or not, as that is irrelevant to the problem to fix.

**Comment by cthoyt** [comment]
I second @Duppils, though my motivation for wanting this information is to disambiguate PyPI user with GitHub user accounts, Wikidata entries, and ORCID identifiers so we in the computational life/natural sciences (and others) can better report on bibliometrics of software

**Comment by miketheman** [comment]
@cthoyt Thanks for the investigation! Would you consider turning this into a pull request? Our dev docs should help get you going. https://warehouse.pypa.io/

**Comment by cthoyt** [comment]
@miketheman yes, in fact I already have the code ready :) will post it later

**Comment by peterk** [comment]
@cthoyt Did you work any more on implementing this? I had a look today and maybe the code from the package html view could be used to load maintainers?

https://github.com/pypi/warehouse/blob/30846f6061d63b8c41a3275d1958efe3749524d8/warehouse/packaging/views.py#L97-L108

**Comment by miketheman** [comment]
Resolved via #19525 https://docs.pypi.org/api/json/#ownership
